### PR TITLE
drivemount: improve code-style

### PR DESCRIPTION
**Pull request by raveit65**
- tabs to spaces
- 4 spaces at the beginning of a line

### CODE REVIEW

File: drivemount/drivemount.c [hunk 1]
```diff
@@ -117,47 +117,47 @@ display_about_dialog (GtkAction *action,
...
+        "title",       _("About Disk Mounter"),
+        "version",     VERSION,
+        "copyright",   _("Copyright \xC2\xA9 2004 Canonical Ltd\n"
+                         "Copyright \xc2\xa9 2012-2020 MATE developers"),
+        "comments",    _("Applet for mounting and unmounting block volumes."),
+        "authors",     authors,
+        "documenters", documenters,
+        "translator-credits", _("translator-credits"),
+        "logo_icon_name",     "media-floppy",
+        NULL);
```
Comment (rbuj) [inline drivemount/drivemount.c:129]: the columns are not aligned

File: drivemount/drive-list.c [hunk 2]
```diff
@@ -39,33 +39,34 @@ static void drive_list_dispose  (GObject *object);
...
+static void add_volume          (DriveList *self,
+                                 GVolume *volume);
+static void remove_volume       (DriveList *self,
+                                 GVolume *volume);
+static void add_mount           (DriveList *self,
+                                 GMount *mount);
+static void remove_mount        (DriveList *self,
+                                 GMount *mount);
+static void queue_relayout      (DriveList *self);
+
```
Comment (rbuj) [inline drivemount/drive-list.c:69]: You don't keep the criteria with the functions that have the drive_list prefix. IMHO, the signatures can be defined using a single line in this case.

File: drivemount/drive-list.c [hunk 3]
```diff
@@ -200,19 +199,20 @@ drive_list_add (GtkContainer *container, GtkWidget *child)
     g_return_if_fail (DRIVE_IS_BUTTON (child));
 
     if (GTK_CONTAINER_CLASS (drive_list_parent_class)->add)
-	(* GTK_CONTAINER_CLASS (drive_list_parent_class)->add) (container,
-								child);
+        (* GTK_CONTAINER_CLASS (drive_list_parent_class)->add) (container,
+                                                                child);
```
Comment (rbuj) [inline drivemount/drive-list.c:203]: what is the benefit of breaking the line here?

File: drivemount/drive-list.c [hunk 4]
```diff
@@ -223,177 +223,174 @@ drive_list_remove (GtkContainer *container, GtkWidget *child)
...
+        g_hash_table_remove (self->volumes, button->volume);
     else
-	g_hash_table_remove (self->mounts, button->mount);
+        g_hash_table_remove (self->mounts, button->mount);
 
     if (GTK_CONTAINER_CLASS (drive_list_parent_class)->remove)
-	(* GTK_CONTAINER_CLASS (drive_list_parent_class)->remove) (container,
-								   child);
+        (* GTK_CONTAINER_CLASS (drive_list_parent_class)->remove) (container,
+                                                                   child);
```
Comment (rbuj) [inline drivemount/drive-list.c:232]: what is the benefit of breaking the line here?

File: drivemount/drivemount.c [hunk 5]
```diff
@@ -36,78 +36,78 @@ static const char drivemount_iid[] = "DriveMountApplet";
...
-	drive_list_set_transparent (drivelist, TRUE);
-	break;
+        case PANEL_NO_BACKGROUND:
+            drive_list_set_transparent (drivelist, FALSE);
+            break;
+
+        case PANEL_COLOR_BACKGROUND:
+        case PANEL_PIXMAP_BACKGROUND:
+            drive_list_set_transparent (drivelist, TRUE);
+            break;
```
Comment (rbuj) [inline drivemount/drivemount.c:93]: this break sentence does nothing

File: drivemount/drive-list.c (hunk 2 above)
Comment (raveit65) [reply drivemount/drive-list.c:69]: That wasn't me, i only fixed the alignment :) 
But, np, i can rewrite it in a single line.

File: drivemount/drive-list.c (hunk 3 above)
Comment (raveit65) [reply drivemount/drive-list.c:203]: This wasn't my idea :)
Any way, i will fix it.

File: drivemount/drive-list.c (hunk 4 above)
Comment (raveit65) [reply drivemount/drive-list.c:232]: same here :)

File: drivemount/drivemount.c (hunk 5 above)
Comment (raveit65) [reply drivemount/drivemount.c:93]: Because of the empty line 89?
Should i remove this line?

Comment (raveit65) [reply drivemount/drivemount.c:93]: Opps, got it.
You meant the break in line 93.
I will  drop it.

**Comment by raveit65** [comment]
All done.

File: drivemount/drivemount.c [hunk 6]
```diff
@@ -117,47 +115,47 @@ display_about_dialog (GtkAction *action,
...
-	"comments",    _("Applet for mounting and unmounting block volumes."),
-	"authors",     authors,
-	"documenters", documenters,
-	"translator-credits", _("translator-credits"),
-	"logo_icon_name",     "media-floppy",
-	NULL);
+                           "title",              _("About Disk Mounter"),
+                           "version",            VERSION,
+                           "copyright",          _("Copyright \xC2\xA9 2004 Canonical Ltd\n"
+                           "Copyright \xc2\xa9 2012-2020 MATE developers"),
```
Comment (rbuj) [inline drivemount/drivemount.c:121]:
```diff
diff --git a/drivemount/drivemount.c b/drivemount/drivemount.c
index 4e4d9378..342ca623 100644
--- a/drivemount/drivemount.c
+++ b/drivemount/drivemount.c
@@ -118,7 +118,7 @@ display_about_dialog (GtkAction *action,
                            "title",              _("About Disk Mounter"),
                            "version",            VERSION,
                            "copyright",          _("Copyright \xC2\xA9 2004 Canonical Ltd\n"
-                           "Copyright \xc2\xa9 2012-2020 MATE developers"),
+                                                   "Copyright \xc2\xa9 2012-2020 MATE developers"),
                            "comments",           _("Applet for mounting and unmounting block volumes."),
                            "authors",            authors,
                            "documenters",        documenters,
```

Comment (raveit65) [reply drivemount/drivemount.c]: Indeed this is one line.
Done.

**Review by rbuj — APPROVED** [review]
LGTM

**Pull request merged**